### PR TITLE
Custom header buttons

### PR DESCRIPTION
**Pull request by ShizukuIchi**
now there's only close button for Error

### CODE REVIEW

**Comment by ghostslayer989** [comment]
Did you not see my PR? Can't figure something out.

File: src/WinXP/index.js [hunk 1]
```diff
@@ -277,7 +284,13 @@ function WinXP() {
   }
   function onClickModalButton(text) {
     dispatch({ type: CANCEL_POWER_OFF });
-    dispatch({ type: ADD_APP, payload: appSettings.Error });
+    dispatch({
+      type: ADD_APP,
+      payload: {
+        ...appSettings.Error,
+        injectProps: { message: 'Something wrong!' },
```
Comment (ghostslayer989) [inline src/WinXP/index.js]:
```suggestion
        injectProps: { message: 'Something's wrong!' },
```

Comment (ghostslayer989) [reply src/WinXP/index.js]: "Something wrong!" Doesn't sound right.

Comment (maracuja-juice) [reply src/WinXP/index.js]: That will break the single quotes. Escape needed or ``

Comment (ghostslayer989) [reply src/WinXP/index.js]: Fixed

Comment (maracuja-juice) [reply src/WinXP/index.js]: I meant backticks for the whole strings.

**Pull request merged**